cloudformation/EditorialProductionMetrics.yml

```diff
@@ -66,6 +66,7 @@ Parameters:
   DBPassword:
     Type: String
     Description: Password for the DB instance
+    NoEcho: true
```

Comment:
This file has been moved out to editorial-tools-platform repo. But this change is there